projects/lib/src/uciengine.cpp

```diff
@@ -768,12 +768,12 @@ void UciEngine::parseLine(const QString& line)
 			m_sendOpponentsName = true;
 		else if (option->name() == "Ponder")
 			m_canPonder = true;
-		else if (option->name().startsWith("UCI_") &&
-			 option->name() != "UCI_LimitStrength" &&
-			 option->name() != "UCI_Elo")
-		{
-			// TODO: Deal with UCI features
-		}
+		//else if (option->name().startsWith("UCI_") &&
+		//	 option->name() != "UCI_LimitStrength" &&
+		//	 option->name() != "UCI_Elo")
+		//{
+		//	// TODO: Deal with UCI features
+		//}
```

Comment:
I think this section needs a bit of justification. Why this? The rationale should be justified in the commit message? What breaks because of the current code?

Also, better just delete this block altogether, rather than comment out. Git history is the proper place for code no longer needed.

Reply:
Warning: Stockfish 202007032109 doesn't have option UCI_ShowWDL                                                                                  
Warning: LCZero v0.26.0-sv-t60-4229-mlh doesn't have option UCI_ShowMovesLeft                                                                       
Warning: LCZero v0.26.0-sv-t60-4229-mlh doesn't have option UCI_ShowWDL                      
                         
For LCZero did in past gave those as command line options, for Stockfish I don't think there is command line option.

Reply:
Either delete this code or add these 2 exceptions too. There might be further ones in future though and then need to add them each time. Which one you think is better?

Reply:
I suggest just deleting this "else if (...) {  }" block. Commented-out code belongs to the era before version control.

Maybe add the following in the commit message:
```
Allow passing options starting with UCI_ to engines. This is needed to send options such as UCI_ShowWDL and UCI_ShowMovesLeft to Lc0.
```